domains/callen.json

```
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "callenflynn",
+    "email": "callen.flynn@proton.me"
+  },
+  "records": {
+    "CNAME": "callenflynn.github.io",
+    "TXT": [
+      "discord=cf151"
+    ]
```

this is not how you add the discord domain connection, read -> https://docs.is-a.dev/guides/discord-verification/
```suggestion
    "CNAME": "callenflynn.github.io"
```